server/controllers/paintings/addPainting.js

```diff
@@ -0,0 +1,45 @@
+const { addPaintingSchema } = require('../../utils/validation');
+const uploadImg = require('../../utils/uploadImg');
+const { addPaintingQuery } = require('../../database/queries');
+
+exports.addPainting = async (req, res, next) => {
+  try {
+    const { title, description, category, property } = req.body;
+    const { id } = req.user;
+    const { paintingImg } = req.files;
```

Comment:
Here you should check if there `req.user` or `req.files` using `if` statement before destructuring
```suggestion
    const { id } = req.user;
    const { paintingImg } = req.files;
```